pyinstrument/profiler.py

```diff
@@ -304,6 +306,13 @@ def output_html(self, timeline: bool = False) -> str:
         """
         return self.output(renderer=renderers.HTMLRenderer(timeline=timeline))
 
+    def write_html(self, path: str | os.PathLike[str], timeline: bool = False):
+        """
+        Writes the profile output as HTML to a file, as rendered by :class:`HTMLRenderer`
+        """
+        file = Path(path)
+        file.write_text(self.output(renderer=renderers.HTMLRenderer(timeline=timeline)))
```

Comment:
The HTML file actually specifies utf-8 via a `<meta charset="utf-8">` tag, so let's write the file as that. It'll also workaround encoding errors on platforms who have strange encodings.

Reply:
Great point! I should have caught that with my diff! Added!